app/1.0.x/health-checks-circuit-breakers.md

```diff
@@ -137,12 +137,17 @@ when a target is healthy or unhealthy. If the `interval` value for either
 is set to zero, the checking is disabled at the corresponding scenario.
 When both are zero, active health checks are disabled altogether.
 
+<div class="alert alert-warning">
+<strong>Note:</strong> Active Health checks currently only support HTTP/HTTPS targets. They
+will self-deactivate when applied to TCP streams.
```

Comment:
"TCP streams" here is ambiguous: HTTP requests after all also go through TCP.
This sentence "will self-deactivate" is very derivative of the actual implementation, and might be a bit misleading wrt the user-visible behavior.

Suggested:

> <strong>Note:</strong> Active health checks currently only support HTTP/HTTPS targets. They do not apply to Upstreams assigned to Services with the `protocol` attribute set to `"tcp"` or `"tls"`.

(capitalization to indicate Kong entities.)